build-system/babel-plugins/babel-plugin-deep-pure/index.js

```diff
@@ -0,0 +1,25 @@
+/** @return {import('@babel/core').PluginObj} */
+module.exports = function () {
```

Comment:
To tree-shake the symbols in `config/urls`, they must:

- Annotate all their 'CallExpressions' and `NewExpressions` with `/* #__PURE__ */`.
- Omit `MemberExpressions`.

This doesn't need to occur through this transform. We could alternatively lint.